src/charm.py

```diff
@@ -572,6 +676,123 @@ def _add_cloud_integrator(
                 },
                 pipelines=["logs"],
             )
+        if tempo_url:
+            self.otel_config.add_exporter(
+                name="otlphttp/cloud-integrator",
+                exporter_config={
+                    "endpoint": tempo_url,
+                    "tls": {"insecure_skip_verify": insecure_skip_verify},
+                    **exporter_auth_config,
+                },
+                pipelines=["traces"],
+            )
```

Comment:
This bit seems to assume `limit: 1`. Should we remove that assumption?
Could be relevant for the hot/cold storage story.